Multiplatform/NowPlaying/NowPlayingButtons.swift

```diff
@@ -113,7 +113,7 @@ struct NowPlayingButtons: View {
         .bold()
         .font(.system(size: 20))
         .frame(height: 45)
-        .padding(.top, 35)
+        .padding(.top, horizontalSizeClass == .regular ? 0 : 35)
```

Comment:
This can be reduced but should not be zero

Reply:
It actually can because the visual padding will still not be zero: 

<img width="381" alt="image" src="https://github.com/rasmuslos/AmpFin/assets/12995396/07aea733-a6fe-4f9e-9453-568984bccd0e">

Reply:
But I think it does look kind of weird to push them so close together, especially when there is so much whitespace between the controls and the cover.

Also I think it might be a good idea to increase the size of the controls / cover again, because the cover is limited by its width and I think it would look better if it was bigger, the controls should adjust to that.

<img width="1611" alt="Bildschirmfoto 2024-04-10 um 10 21 12" src="https://github.com/rasmuslos/AmpFin/assets/49640416/90df8c5c-c8e2-41ce-b32c-34cdbcd2f9e4">

Reply:
Let me think about it again and try to find a sane width so that the spacing would not be this weird.

Reply:
![Simulator Screenshot - iPad Pro (12 9-inch) (6th generation) (16GB) - 2024-04-10 at 10 38 25](https://github.com/rasmuslos/AmpFin/assets/49640416/46cdd370-7404-4362-8cd3-8bc5edcb8025)

This is what I have come up with. It has equal padding of 40 units and the gap is 80. If focus of the lyrics section is at the center this could be pretty good. Although I still think that the controls should have a little more spacing.

Reply:
Looks good to me. The current spacing of controls is hand-measured by myself from the Apple Music App, but the spacing between cover image and the controls appears to be significant smaller on Apple Music. Probably we should put the controls more closer and make the bottom just a spacer?